imap_processing/tests/swe/test_swe_l1a_science.py

```diff
@@ -126,36 +97,37 @@ def test_swe_science_algorithm(decom_test_data):
 
     # Test data has n packets, therefore, SPIN_PHASE should have that same length.
     spin_phase = processed_data["SPIN_PHASE"]
-    expected_length = 4
+    expected_length = 29
     assert len(spin_phase) == expected_length
 
 
-def test_decompress_counts(decom_test_data):
-    """Test decompress counts."""
-    test_data_path = imap_module_directory / "tests/swe/decompressed"
-    filepaths = [
-        "20230927173253_1st_quarter_decompressed.csv",
-        "20230927173308_2nd_quarter_decompressed.csv",
-        "20230927173323_3rd_quarter_decompressed.csv",
-        "20230927173238_4th_quarter_decompressed.csv",
-    ]
-    decompressed_data = swe_science(decom_test_data)
-
-    for index in range(len(filepaths)):
-        instrument_decompressed_counts = pd.read_csv(
-            test_data_path / f"{filepaths[index]}", index_col="Index"
-        )
-
-        assert (
-            decompressed_data["QUARTER_CYCLE"].data[index]
-            == decom_test_data[index].data["QUARTER_CYCLE"].raw_value
-        )
-        sdc_decompressed_counts = (
-            decompressed_data["SCIENCE_DATA"].data[index].reshape(180, 7)
-        )
-
-        for i in range(7):
-            cem_decompressed_counts = instrument_decompressed_counts[
-                f"CEM {i+1}"
-            ].values
-            assert np.all(sdc_decompressed_counts[:, i] == cem_decompressed_counts)
+# TODO: Get validation data for decompressed counts
```

Comment:
Decorate this with `@pytest.skip("Don't have validation data yet")` instead of commenting it out?